crates/matrix-sdk/src/sliding_sync/error.rs

```diff
@@ -53,4 +54,14 @@ pub enum Error {
         /// The original `JoinError`.
         error: JoinError,
     },
+
+    /// No Olm machine.
+    #[cfg(feature = "e2e-encryption")]
+    #[error("The Olm machine is missing")]
+    NoOlmMachine,
+
+    /// An error occurred during a E2EE operation.
+    #[cfg(feature = "e2e-encryption")]
+    #[error(transparent)]
+    OlmError(#[from] OlmError),
```

Comment:
I think you only need the `CryptoStoreError`?

Reply:
Yup. Done.